functions/signup_password.js

```diff
@@ -1,34 +1,17 @@
-import { inputPassword } from "../tags.js";
-import { removeRedMessage, createRedMessage } from "./error_message.js";
+import { errorMessages } from "../tags.js";
+import { removeRedMessage, createRedMessage } from "./common.js";
 
-let passwordValid = "";
-
-function printEmpty() {
-  createRedMessage(inputPassword, "비밀번호를 입력해주세요.");
-}
-
-function printWorng() {
-  createRedMessage(
-    inputPassword,
-    "비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.",
-  );
-}
+export let passwordValid = false;
 
 // 비밀번호 유효성을 검증하는 함수
-function enterPasswordMessage() {
+export function enterPasswordMessage(input) {
   const passwordFormat = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
-  const passwordValue = inputPassword.value;
-  passwordValid = false;
+  const passwordValue = input.value;
 
-  removeRedMessage(inputPassword);
+  removeRedMessage(input);
 
-  if (!passwordValue) {
-    printEmpty();
-  } else if (!passwordFormat.test(passwordValue)) {
-    printWorng();
-  } else {
-    passwordValid = true;
-  }
+  if (!passwordValue) createRedMessage(input, errorMessages.enterPassword);
+  else if (!passwordFormat.test(passwordValue))
+    createRedMessage(input, errorMessages.checkFormPassword);
+  else passwordValid = true;
```

Comment:
이거는 사람마다 의견이 다르기는 한데 저는 괄호로 묶어주는 것이 더 가독성이 좋다고 생각합니다~ 

```suggestion
  if (!passwordValue) {
    createRedMessage(input, errorMessages.enterPassword);
  } else if (!passwordFormat.test(passwordValue)) {
    createRedMessage(input, errorMessages.checkFormPassword);
  } else {
    passwordValid = true;
  }
```